aya/src/maps/mod.rs

```diff
@@ -232,36 +234,44 @@ fn maybe_warn_rlimit() {
 /// eBPF map types.
 #[derive(Debug)]
 pub enum Map {
-    /// A [`Array`] map
+    /// A [`Array`] map.
```

Comment:
:heart:

Reply:
Not a native speaker but shouldn't this be ```An [`Array`] map.``` ?

Reply:
ha indeed, missed this one in the update.

Reply:
fixed!